sdk/core/core-sse/test/matrix.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
we have a tracking bug for de-duplicating all these copies of `matrix` right?

Reply:
This should be called out in https://github.com/Azure/azure-sdk-for-js/issues/28475 for upgrading `test-utils`